docs/rabbitmq.config.example

```diff
@@ -246,6 +246,10 @@
    %%
    %% {memory_monitor_interval, 2500},
 
+   %% The total memory available can be calculated from the OS resources
+   %% - default option - or provided as a configuration parameter:
+   %% {total_memory_available_override_value, "5000MB"},
```

Comment:
Units are specified just above: https://github.com/rabbitmq/rabbitmq-server/blob/1b976358b62db552f70a0f6c273431895e3cddb0/docs/rabbitmq.config.example#L230. The problem is that we report the wrong unit back to the user here: https://github.com/rabbitmq/rabbitmq-server/blob/master/src/vm_memory_monitor.erl#L240, using MB when the conversion applied for that log message is MiB.